mysql-test/suite/sysschema/r/v_innodb_buffer_stats_by_table.result

```diff
@@ -8,7 +8,11 @@ pages	bigint(21)	NO		0
 pages_hashed	bigint(21)	NO		0	
 pages_old	bigint(21)	NO		0	
 rows_cached	decimal(44,0)	YES		NULL	
-SELECT * FROM sys.innodb_buffer_stats_by_table;
+CREATE TABLE t1(a INT PRIMARY KEY) ENGINE=InnoDB;
```

Comment:
I suppose you wanted to test the state when `pages_hashed` and `pages_old` are zero and non-zero. Why don't do this in `v_innodb_buffer_stats_by_(schema|table)` tests? I would propose to move the case from `innochecksum.test` to `v_innodb_buffer_stats_by_(schema|table).test`.